src/components/space/ActionsMap.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import GoogleMapReact from 'google-map-react';
+import useSupercluster from 'use-supercluster';
+import spaceData from '../../data/sample.json';
+
+const useStyles = makeStyles(() => ({
+  clusterMarker: {
+    color: '#fff',
+    background: '#5050d2',
+    borderRadius: '50%',
+    padding: 10,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+}));
+
+const Marker = ({ children }) => children;
+
+function ActionsMap() {
+  const classes = useStyles();
+  const [bounds, setBounds] = useState(null);
+  const [zoom, setZoom] = useState(5);
+
+  const points = spaceData.map((action) => ({
+    type: 'Feature',
+    properties: { cluster: false, actionId: action._id },
+    geometry: {
+      type: 'Point',
+      coordinates: [action.geolocation.ll[1], action.geolocation.ll[0]],
+    },
+  }));
+
+  const { clusters } = useSupercluster({
+    points,
+    bounds,
+    zoom,
+    options: { radius: 75, maxZoom: 10 },
+  });
+
+  return (
+    <Container style={{ width: '100%', height: '100%' }}>
```

Comment:
Not sure we need `Container`. Perhaps we can just use `<>`.

Reply:
You're right. Removed the Container and used a Fragment and it even looks better.